grails-app/controllers/org/broadinstitute/orsp/api/IssueReviewController.groovy

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.orsp.api
+
+import com.google.gson.Gson
+import grails.converters.JSON
+import groovy.util.logging.Slf4j
+import org.broadinstitute.orsp.AuthenticatedController
+import org.broadinstitute.orsp.Issue
+import org.broadinstitute.orsp.IssueReview
+import org.broadinstitute.orsp.IssueReviewService
+
+@Slf4j
+class IssueReviewController extends AuthenticatedController {
+
+    IssueReviewService issueReviewService
+
+    def save() {
+        Gson gson = new Gson()
+        IssueReview issueReview = gson.fromJson(gson.toJson(request.JSON), IssueReview.class)
```

Comment:
This is fun to read: `gson.fromJson(gson.toJson(request.JSON)`  :-)
Maybe we can extract that out into a method so it isn't so jarring on the eyes?
```
private IssueReview parseIssueReview(String json) {
    new Gson().fromJson(json, IssueReview.class)
}
```
and then here:
```
IssueReview issueReview = parseIssueReview(gson.toJson(request.JSON)
```

Reply:
Thx Greg! 
Tested and worked well. I'll merge this.